### PR TITLE
Add action for exporting SignupTarget participants to CSV

### LINKED ISSUE #9: Export SignupTarget's participants list in CSV

### PR DESCRIPTION
**Pull request by igordavydsson**
Closes #9

### CODE REVIEW

File: signups/admin.py [hunk 1]
```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
     fields = ('name', 'identifier')
     prepopulated_fields = {'identifier': ('name',)}
     inlines = (SignupInline,)
+    actions = ["export_participants"]
+
+    def export_participants(self, request, queryset):
+        if queryset.count() > 1:
+            return self.message_user(request, _(u"Valitse vain yksi Signup Target"), messages.ERROR)
```
Comment (tuomas777) [inline signups/admin.py]: Source language for translated fields should be English.

File: signups/admin.py [hunk 2]
```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
...
+
+        signup_target = queryset.first()
+        participants = signup_target.users.all()
```
Comment (tuomas777) [inline signups/admin.py]: This should exclude cancelled sign-ups, ie. `signup_target.users.exclude(cancelled_at=None))`

File: signups/admin.py [hunk 3]
```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
...
+
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename=orders.csv'
```
Comment (tuomas777) [inline signups/admin.py]: Not sure why the field is named `orders` :) Also, maybe include signup target name and some kind of rough timestamp to the filename as well?

File: signups/admin.py [hunk 4]
```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
...
+        export_participants_as_csv(participants, response)
+        return response
+    export_participants.short_description = _("Vie osallistujen lista CSV-tiedostoon")
```
Comment (tuomas777) [inline signups/admin.py]: Should be `osallistujien`. Actually maybe "Lataa ilmoittautuneiden lista CSV-tiedostona." would be better.

**Review by tuomas777 — CHANGES_REQUESTED** [review]

**Comment by igordavydsson** [comment]
@tuomas777 good points! fixed things up.

**Review by tuomas777 — APPROVED** [review]
👍

**Pull request merged**